lib/alerts_viewer_web/router.ex

```diff
@@ -28,6 +28,7 @@ defmodule AlertsViewerWeb.Router do
 
     live("/bus", BusLive)
     live("/alerts-to-close", AlertsToCloseLive)
+    live("/internal-alerts-to-close", InternalAlertsToCloseLive)
```

Comment:
question: The ticket suggests the route `/open-delay-alerts`—what do you think of that wording (probably for the page title and menu as well)? My first thought is maybe it might be slightly more accurate. And one way on another having more distinct paths between the 2 pages would probably be nicer.

Reply:
I missed that in the ticket, I agree it's nice having it be more distinct.